README.md

```diff
@@ -364,12 +363,15 @@ Share your apps here! Submit a pull request!
 
 * **Tailwinder** This an open-source complete community website project created for Appwrite hackathon. [Source Code](https://github.com/MooseSaeed/Tailwinder) [Demo](https://dev.to/moose_said/introducing-tailwinder-home-for-tailwind-fans-powered-with-appwrite-4c0o) [Video Demo](https://vimeo.com/manage/videos/709162501)
 
+* **Appwrite + Vue Podcast Transcript** A simple Podcast Transcript app built with Appwrite, Deepgram and Vue. [Source Code](https://github.com/timothee-durand/appwrite-transcript)
+
+### Nuxt
+* **[nuxt3-appwrite package](https://github.com/YsarocK/nuxt-appwrite)** A package to simply implement Appwrite in your Nuxt 3 project. Composable, Routes guards & no config needed. [Documentation](https://nuxt-appwrite.etiennemoureton.fr)
```

Comment:
```suggestion
* **nuxt3-appwrite package** A package to simply implement Appwrite in your Nuxt 3 project. Composable, Routes guards & no config needed. [Source Code](https://github.com/YsarocK/nuxt-appwrite), [Documentation](https://nuxt-appwrite.etiennemoureton.fr)
```

Suggesting this change to ensure consistency of format with other submissions in the file